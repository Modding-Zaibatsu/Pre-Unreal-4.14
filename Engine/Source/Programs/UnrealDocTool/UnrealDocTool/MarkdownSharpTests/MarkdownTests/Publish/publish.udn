1. Item 1.
[PUBLISH:Rocket]
1. Item 2R.
[/PUBLISH]
[PUBLISH:Licensee]
1. Item 2.
[/PUBLISH]
1. Item 3.


1. Item 1.
[PUBLISH:Licensee]
1. Item 2.
[/PUBLISH]
[PUBLISH:Rocket]
1. Item 2R.
[/PUBLISH]
1. Item 3.


1. Item 1.
[PUBLISH:Licensee]
1. Item 2.
[/PUBLISH]
1. Item 3.


    [PUBLISH:Licensee]
    Licensee code block
    [/PUBLISH]
    [PUBLISH:Rocket]
    Rocket code block
    [/PUBLISH]
    [PUBLISH:Licensee]
    Licensee 2 code block
    [/PUBLISH]


[OBJECT:TopicList]
    [PARAM:icon]
        Test icon.
    [/PARAM]
    [PARAM:title]
        Test title.
    [/PARAM]
    [PARAM:description]
    [PUBLISH:Licensee]
        Licensee paragraph fragment.
    [/PUBLISH:Licensee]
    [PUBLISH:Rocket]
        Rocket paragraph fragment.
    [/PUBLISH:Rocket]
    [/PARAM]
    [PARAM:links]
        Test links.
    [/PARAM]
[/OBJECT]
