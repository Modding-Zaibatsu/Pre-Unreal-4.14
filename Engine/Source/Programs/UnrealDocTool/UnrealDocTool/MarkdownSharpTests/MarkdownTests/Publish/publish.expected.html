<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        <ol>
            <li><p>Item 1.</p></li>
            <li><p>Item 2.</p></li>
            <li><p>Item 3.</p></li>
        </ol>
        
        <ol>
            <li><p>Item 1.</p></li>
            <li><p>Item 2.</p></li>
            <li><p>Item 3.</p></li>
        </ol>
        
        <ol>
            <li><p>Item 1.</p></li>
            <li><p>Item 2.</p></li>
            <li><p>Item 3.</p></li>
        </ol>
        
        <pre class="prettyprint"><code>Licensee code block
Licensee 2 code block</code></pre>
        
        <div class="topic">
	        <div class="icon">
		        <p>Test icon.</p>
	        </div>
	        <div class="wrapper transparent">
		        <div class="largetitle">
			        <p>Test title.</p>
		        </div>
		        <p>Licensee paragraph fragment.</p>
		        <div class="arrowlist">
			        <p>Test links.</p>
		        </div>
	        </div>
        </div>
    </body>
</html>
